policy-controller/k8s/index/src/tests.rs

```diff
@@ -30,14 +32,18 @@ async fn incrementally_configure_server() {
         detect_timeout,
     );
 
+    let mut pods = mock(idx.clone(), crate::pod::index);
+    let mut servers = mock(idx.clone(), crate::server::index);
+    let mut server_authzs = mock(idx, crate::server_authorization::index);
```

Comment:
rather than call into module-private functions, we drive a channel through the main `index` entrypoint